src/clims/models/step.py

```diff
@@ -0,0 +1,32 @@
+from __future__ import absolute_import, print_function
+
+from django.db import models
+from sentry.db.models import (FlexibleForeignKey, sane_repr)
+from clims.models.extensible import ExtensibleModel, ExtensibleVersion
+
+
+class Step(ExtensibleModel):
```

Comment:
I wonder still if we need a django model for this or not. We already have a django model that says that version x of a certain plugin is installed, so in that way we have traceability for this. We might want to discuss this via a chat.

Reply:
Yes. This inheritance was kind of a leftover since I was experimenting back and forth. I'll remove this one.